src/Symfony/Component/Mime/Encoder/Base64ContentEncoder.php

```diff
@@ -32,11 +32,8 @@ public function encodeByteStream($stream, int $maxLineLength = 0): iterable
             throw new RuntimeException('Unable to set the base64 content encoder to the filter.');
         }
 
-        if (stream_get_meta_data($stream)['seekable'] ?? false) {
-            rewind($stream);
```

Comment:
Rewinding is already done before the call, and encoders shouldn't rewind as that makes them less useful in the generic case.